app/actions/index.js

```diff
@@ -32,11 +37,11 @@ export const createDat = () => dispatch => {
   addDat({ path })(dispatch)
 }
 
-export const addDat = ({ key, path, paused, ...opts }) => dispatch => {
+export const addDat = ({ key, path, ...opts }) => async dispatch => {
```

Comment:
we need to be able to add immediately paused dats, because when you pause a dat and restart the app it's still paused

Reply:
`loadFromDisk` loads `pausedDats` before adding all dats dats, this way if they are paused at startup.